Add some piece for offline prepare support. (#310)
microsoft/onnxruntime-inference-examples: c_cxx/QNN_EP/mobilenetv2_classification/README.md

    - a. Set qnn_context_cache_enable to 1 and run with QDQ model.
    - b. The first run will generate the context binary model (Default file name is model_file_name.onnx_qnn_ctx.onnx if qnn_context_cache_path is not set.).
    - c. Use the generated context binary model (mobilenetv2-12_quant_shape.onnx_qnn_ctx.onnx) for inference going forward. (No need the QDQ model, no need to set the qnn_context_cache_enable).

>     - Notes: The QNN context binary is embeded inside the Onnx model by default. You can also set QNN EP option qnn_context_embed_mode = 0, so that the it will generate the QNN context as a separate file and put the relative path in the Onnx model. It' useful if the QNN context binary size exceeds 2GB limit of protobuf.

Reviewer: ```suggestion
    - Notes: The QNN context binary is embedded within the ONNX model by default. Alternatively, set the QNN EP session option qnn_context_embed_mode to 0 in order to generate the QNN context binary as a separate file and embed the file's relative path in the ONNX model. This is necessary if the QNN context binary size exceeds protobuf's 2GB limit.
```

	  - python gen_qnn_ctx_onnx_model.py -b libmobilenetv2-12.serialized.bin -q mobilenetv2-12_net.json
    - c. Create ONNX Runtime session with the model generated from step b.
	- d. Run the model with quantized input data. The output also need to be dequantized. This is because QNN quantized model use quantized data type for model inputs & outputs. More details refer to QuantizedData & DequantizedData in [main.cpp](https://github.com/microsoft/onnxruntime-inference-examples/blob/main/c_cxx/QNN_EP/mobilenetv2_classification/main.cpp). Also the input image is NHWC layout for QNN converted model.

>     - Notes: Set --disable_embed_mode to gen_qnn_ctx_onnx_model.py, so that it will generate the Onnx model with the relative path to the QNN context bianry file. It' useful if the QNN context binary size exceeds 2GB limit of protobuf.

Reviewer: ```suggestion
    - Notes: Call gen_qnn_ctx_onnx_model.py with --disable_embed_mode to generate the ONNX model with the relative path to the QNN context binary file. This is necessary if the QNN context binary size exceeds protobuf's 2GB limit.
```
